### DISCUSSION TITLE
All locales except english cause undefined on field. Use custom locale inside project folder also not working

### DISCUSSION THREAD

**Discussion by jay2success** [vueform/vueform#262] [Questions]
Hello,

All locales except english when validated shows "undefined" on the field.
When i check the locale file, the mistakes comes from using :Attribute instead of :attribute.
When i tried to use custom locale inside project folder, when validated, it doesn't show anything.
Please help.
Thanks.

**Comment by adamberecz** [comment]
Thanks for reporting—fixed in `1.10.0`
